python/cuml/ensemble/randomforestregressor.pyx

```diff
@@ -507,24 +509,26 @@ class RandomForestRegressor(Base):
 
         return ctypes.c_void_p(mod_handle).value
 
-    def _concatenate_treelite_handle(self, treelite_handle):
+    def _concatenate_treelite_handles(self, treelite_handles):
         cdef ModelHandle concat_model_handle = NULL
         cdef vector[ModelHandle] *model_handles \
             = new vector[ModelHandle]()
         cdef uintptr_t mod_ptr
-        for i in treelite_handle:
+        for i in treelite_handles:
             mod_ptr = <uintptr_t>i
             model_handles.push_back((
                 <ModelHandle> mod_ptr))
 
+        # Who frees the old treelite model here?
         concat_model_handle = concatenate_trees(deref(model_handles))
         cdef uintptr_t concat_model_ptr = <uintptr_t> concat_model_handle
         self.treelite_handle = concat_model_ptr
-        cdef vector[unsigned char] pbuf_mod_info = \
```

Comment:
Confused about why we do not obtain the model_pbuf_bytes for the concatenated model in rf regression but we do so for rf classifier.

Reply:
I probably just didn't update the regressor correctly during the various merges - will make it look like classifier

Reply:
Actually, I think regressor was the correct style. I think it's better to have only one place where we're caching/updating protobuf, and that will automatically be handled in get_protobuf_bytes